README.md

```diff
@@ -6,8 +6,15 @@
 Describe why your topic is important and what you want to share with your audience
 
 ## Requirements
-Do not forget to indicate Python version and any other tools
-+ add requirements.txt or conda.yml or docker image or Binder/Google Collab link
+PyLadies encourage the usage of [uv](https://astral.sh/blog/uv) for dependency management. You can get uv by running the following command in your shell:
+```bash
+curl -LsSf https://astral.sh/uv/install.sh | sh
+```
+To get started, open the `pyproject.toml` file and set the required python version. The pre-selected version 3.8 is generally a safe choice for most use cases.
+
+After you have specified the python version, you can create a virtual environemnt with `uv venv` and add packages with `uv add <package>`. Before the workshop, you can generate a requirements.txt file by running `uv export > requirements.txt`.
+
+If you are not familiar with uv or are in a hurry, feel free to work with your familiar dependency management system and the PyLadies will handle the `pyproject.toml` for you.
```

Comment:
Please move the rest of the info (lines 13-17 to the Usage)

Reply:
Do we really want to handle pyproject.toml for them?

Reply:
I moved the section to the Usage section, overwriting what's currently there. 

Regarding the `pyproject.toml` file, I wouldn't say we "want" to handle it for them, but I was thinking to give the option if people are not familiar with it.
But then again, it's so simple, that I removed that part for now.